### DISCUSSION TITLE
A little help needed for shareservice/dl.php settings

### DISCUSSION THREAD

**Discussion by jaydee73** [photobooth-app/photobooth-app#346] [Q&A]
Hi guys,

I am a little bit lost regarding the right settings for the dl.php file.

I have a running website (German hoster Netcup). Let's say it's https://www.mywebsite.de. I've uploaded the edited dl.php to the site. It is reachable via www.mywebsite.de/dl.php?action=info, where I get a valid response. There is a folder "uploads" created in this directory, also a sqlite db file. So I assume the setup is correct.

Nevertheless the qrcode-upload doesn't work. I'll get a timeout when I access the website through the qr-code (the url from the qr-code indeed seems to be correct).

I think maybe the line ` $WORK_DIRECTORY = __DIR__ . "/uploads";` could be the problem. I don't really know what should be the correct path here?

I do see lots of the following error(s) in the php-error.log:

`23-Aug-2024 18:46:02 Europe/Berlin] PHP Warning:  is_dir(): open_basedir restriction in effect. File(/uploads) is not within the allowed path(s):`

I have already tried different paths as working directory, all with the same result.

Every kind of help to troubleshoot this would be very nice!

Regards,
JD

**Comment by Rouliooooo** [comment]
Hey !
Maybe a little late to answer but might help other users : try by elevating rights on the folder your dl.php is stored.
Seems that your server doesn't have necessary permissions.

Could be something like :

```
chown www-data:www-data /var/www/html/dl.php
chmod 644 /var/www/html/dl.php
```
